receiver/httpcheckreceiver/metadata.yaml

```diff
@@ -0,0 +1,32 @@
+name: httpcheckreceiver
+
+resource_attributes:
+
+attributes:
+  http.url:
+    description: Full HTTP request URL.
+    type: string
+  http.status_code:
+    description: HTTP response status code
+    type: int
+  http.method:
+    description: HTTP request method
+    type: string
+
+metrics:
+  httpcheck.status:
+    description: 1 if the endpoint resulted in status http.status_code, otherwise 0.
```

Comment:
200? 2xx?

Reply:
updated the description to clarify that that 1 is if the check return a status code within a specific class